.circleci/config.yml

```diff
@@ -38,67 +50,127 @@ commands:
           command: |
             choco install dotnet-sdk --version <<parameters.version>> -y --no-progress
           name: Install .NET Core Latest
-  
+
 jobs:
   build:
-    executor:
-      name: win/default
-      size: medium
-    # macos:
-    #   xcode: "11.3.0"
-    # Let's keep this disabled until we figure out the macOS build.
+    parameters:
+      os:
+        type: executor
+    executor: << parameters.os >>
     steps:
-      - run: git config --global core.symlinks true # https://github.com/git-for-windows/git/wiki/Symbolic-Links 
+      - when:
+          condition: 
+            equal: [ *windows, << parameters.os >>]
+          steps:
+            - run:
+                command: |
+                  git config --global core.symlinks true # https://github.com/git-for-windows/git/wiki/Symbolic-Links 
+                  nvm install lts
+                  rm "/c/Program Files/nodejs" -rf
+                  nvm use 16.16.0
       - checkout-update-submodules:
           paths: "NineChronicles.Headless"
       - yarn-install
-      - install-dotnet
-      - run: 
-          command: |
-            [Environment]::SetEnvironmentVariable("APV", (./scripts/make-apv.ps1), "User")
-          name: Generate APV
-      - run: 
-          command: |
-            set -ex
-            mkdir -p dist
-            curl -o "dist/config.json" 'https://download.nine-chronicles.com/9c-launcher-config.json'
-          shell: bash.exe
-          name: Copy config.json from the last release
-      - run: yarn sign-apv
+      - when:
+          condition:
+            equal: [ *macos, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  brew install --cask dotnet-sdk
+                  curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+                  sudo installer -pkg AWSCLIV2.pkg -target /
+                name: Install .NET Core, AWS
+      - when:
+          condition:
+            equal: [ *linux, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
Using [`dotnet-install` script](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script) could be more beneficial in this case since we don't need to have a separate macOS/Linux installation process.

Reply:
Um, we actually need to have separate macOS/Linux installation process, because we have AWS CLI integrated on ubuntu image, but there's no AWS or .NET SDK integrated in macOS image. so, for the ease of process (i.e. it is separated after all, and dotnet-install sometimes didn't worked for some reason) I used each package managers to install .NET SDK.